Seungsu/app.js

```diff
@@ -35,11 +38,24 @@ app.get("/ping", (req, res) => {
   res.json({ message: "pong" });
 });
 
-const server = http.createServer(app);
-const PORT = process.env.PORT;
+app.post("/signup", async (req, res, next) => {
+  const { name, email, password, profile_image } = req.body;
```

Comment:
```suggestion
  const { name, email, password, profileImage } = req.body;
```
- JavaScript에서는 기본적으로 `snake_case`가 아닌 `CamelCase`로 작성합니다.
- 또한 snake_case, CamelCase를 뒤죽박죽 섞어서 사용하면 안됩니다.
- 하나의 컨벤션으로 통일해서 작성해 주세요.

Reply:
네, CamelCase로 통일했습니다.